pkg/vm/context.go

```diff
@@ -176,44 +177,44 @@ func (c *Context) ScriptHash() util.Uint160 {
 	return c.scriptHash
 }
 
-// Value implements StackItem interface.
+// Value implements Item interface.
```

Comment:
`stackitem.Item`
Not so important though

Reply:
Fixed